hatchery/authz.go

```diff
@@ -233,14 +233,14 @@ var isUserAuthorizedForPayModels = func(userName string, allowedPayModels []stri
 var isUserAuthorizedForResourcePaths = func(userName string, accessToken string, resourcePaths []string) (bool, error) {
 	Config.Logger.Printf("DEBUG: Checking user '%s' access to resource paths %v (service 'jupyterhub', method 'launch')", userName, resourcePaths)
 
-	body := "{ \"requests\": ["
+	body := fmt.Sprintf("{\"user\": {\"token\": \"%s\"}, \"requests\": [", accessToken)
```

Comment:
Wondering why do we need to change this? Is sending the `access token` as a Bearer token not working anymore?

Reply:
it never worked, this arborist endpoint expects the token in the body